src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetsContainer/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, ResponsiveContext } from 'grommet';
+import React from 'react';
+
+interface DashboardVideoLiveWidgetsContainerProps {
+  componentsList: React.ReactNode[];
+}
+
+export const DashboardVideoLiveWidgetsContainer = ({
+  componentsList,
+}: DashboardVideoLiveWidgetsContainerProps) => {
+  const size = React.useContext(ResponsiveContext);
+  let nbrOfColumns: number;
+  switch (size) {
+    case 'small':
+      nbrOfColumns = 1;
+      break;
+    case 'medium':
+      nbrOfColumns = 2;
+      break;
+    default:
+      nbrOfColumns = 3;
+  }
+
+  if (componentsList.length === 0) {
+    return null;
+  }
+
+  return (
+    <Box direction="row" gap="10px" pad={{ horizontal: '10px' }}>
+      {[...Array(nbrOfColumns)].map((_, indexColumn) => (
+        <Box
+          direction="column"
+          gap="10px"
+          key={indexColumn}
+          width={`${100 / nbrOfColumns}%`}
+        >
+          {componentsList
+            .filter((__, indexComponent) =>
+              indexComponent < nbrOfColumns
+                ? indexComponent === indexColumn
+                : indexComponent % nbrOfColumns === indexColumn,
+            )
+            .map((c, index) => (
```

Comment:
`c` is not relevant, use a variable name guessing what it contains to ease the read please.

Reply:
done in [d011e2d704](https://github.com/openfun/marsha/pull/1463/commits/d011e2d704cf0ae1f5cd4a2eecc9d304aa1e096b)